wpinet/src/main/native/cpp/PortForwarder.cpp

```diff
@@ -50,6 +51,7 @@ void PortForwarder::Add(unsigned int port, std::string_view remoteHost,
   m_impl->runner.ExecSync([&](uv::Loop& loop) {
     auto server = uv::Tcp::Create(loop);
     if (!server) {
+      wpi::print(stderr, "PortForwarder: Creating server failed\n");
```

Comment:
we could throw an exception here which would avoid the need to do a manual return as well if performance isn't too much of an issue